tests/test_nodes.py

```diff
@@ -330,3 +330,62 @@ def test_create_bool_with_extra(client, authenticate):  # pylint: disable=unused
     assert check_response.status_code == 200, response.content
     assert check_response.json()['extras']['extra_one'] == 'value_1'
     assert check_response.json()['extras']['extra_two'] == 'value_2'
+
+
+def test_get_full_types(default_nodes, client):
```

Comment:
```suggestion
def test_get_full_types(default_computers, example_processes, default_groups, default_nodes, client):
```

Better to test with as many node types as we can. let's use all the fixtures.